specifications/api/swagger.yaml

```diff
@@ -5244,7 +5322,10 @@ paths:
       description: |
         <p align="right">status: <strong>⚠ under development</strong></p>
 
-        Create a shared wallet from an account public key, its index and script templates.
+        Create a shared wallet from an account public key and script
+        template index.
+
+        **TODO:** ADP-866 is it also restored? If not, when?
```

Comment:
Would it be possible to describe what happens when the shared wallet is created? Is it restored immediately, or must further API requests be made, etc, etc.

Reply:
done